wikidp/sparql.py

```diff
@@ -42,6 +42,21 @@
     ORDER BY ASC(xsd:integer(STRAFTER(STR(?property), 'P')))
 """
 
+ALL_REFERENCE_PROPERTIES = """
```

Comment:
@emulatingkat  this sparql query fetches all properties with instance of ["Wikidata property to indicate a source (Q18608359)"](https://www.wikidata.org/wiki/Q18608359)
Let me know if you'd like the reference set to be a different property subset and we can drop in here!